t/lib/t/MusicBrainz/Server/Controller/User/Edit.pm

```diff
@@ -57,13 +51,76 @@ $mech->get_ok($verify_email_path);
 $mech->content_contains("Thank you, your email address has now been verified!");
 
 $mech->get('/user/new_editor');
-$mech->content_contains('http://example.com/~new_editor/');
-$mech->content_contains('hello world!');
 $mech->content_contains('new_email@example.com');
 
 
 };
 
+test 'Limited users cannot edit website and biography' => sub {
```

Comment:
Do we need the URL format/date checking tests in there too? They seem to duplicate what's above and are unrelated to the description, but I could be missing something.